src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/controller/OrchestrationSessionsController.kt

```diff
@@ -124,8 +124,14 @@ class OrchestrationSessionsController(private val visitSchedulerSessionsService:
       description = "Defaults to true if not passed. If true, will not return visit times that clash with higher priority legal or medical appointments.",
     )
     withAppointmentsCheck: Boolean? = true,
+    @RequestParam(value = "excludedApplicationReference", required = false)
+    @Parameter(
+      description = "The current application reference be to exclude from capacity count and double booking",
```

Comment:
Small update, change the description to:

"The current application reference to be excluded from capacity count and double booking"